_summaries/gropp2020implicit_1.md

```diff
@@ -1,19 +1,55 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{gropp2020implicit}}
 # Please fill out info below
-author: # Your GitHub id
+author: mmcinnestaylor # Your GitHub id
 score: # How did you like this paper 0(dislike) to 10(love)
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## Core idea?  
+* 2D and 3D Shapes can be represented as level sets of neural networks.    
+* Neural shape representations can be learned from raw data by using implicit geometric regularization.  
+* Implicit neural representations can be achieved without 3D supervision and/or a direct loss on the zero level set.     
```

Comment:
Add an additional point about Eikonal equations here (top-left of page 3) and that the authors used a loss with Eikonal term to find functions that can solve Eikonal equations. Also, consider mentioning the property that solutions of Eikonal equations have: unit normals (gradients).